tools/package-lock.json

```diff
@@ -0,0 +1,837 @@
+{
+  "name": "template-schema-validation-tools",
```

Comment:
>"name": "template-schema-validation-tools", [](start = 2, length = 43)

do we need this file?

Reply:
Hmm not sure why it got included. I think this is a temp file. Removed.